src/js/geolocation.js

```diff
@@ -0,0 +1,75 @@
+const btnPos = document
+    .querySelector('.leftAsideBox__findCity--button')
+    .addEventListener('click', getLocation);
+const btnWeather = document
+    .querySelector('.leftAsideBox__ChooseCity--button')
```

Comment:
to nie jest dobre uzycie bema, to nie jest bem wlasciwie

Reply:
Nazwy klas przyjąłem takie jakie są w index.html jeśli się zmienią to je poprawię.